infra/charts/feast/values.yaml

```diff
@@ -51,12 +52,15 @@ feast-core:
   # to the client. These instances of Feast Serving however can still use
   # the same shared Feast Core.
   enabled: true
-  # jvmOptions are options that will be passed to the Java Virtual Machine (JVM) 
+  # Specify what image tag to use. Keep this consistent for all components
+  image:
+    tag: "0.4.3"
```

Comment:
I think we also need to remove the existing image tag from the helm charts. I believe it is currently hardcoded.